sdk/storage/CMakeLists.txt

```diff
@@ -119,7 +119,7 @@ if(BUILD_STORAGE_SAMPLES)
 endif()
 
 if(BUILD_TESTING)
-  add_subdirectory(test)
+  #add_subdirectory(test)
```

Comment:
You probably did not intend to commit this change.

Reply:
Yeah, you can revert it.
CI is currently not running Storage unit tests (only compiling it).